js/ui/floating_action_button_repaint.js

```diff
@@ -0,0 +1,15 @@
+import { repaint } from "./speed_dial_action/speed_dial_main_item";
```

Comment:
I personally don't like the place where you put this file. Why not to move it to speed_dial_action folder and require it from there in widgets-base.js? Also, the file should be named `repaint_floating_action_button.js`.